osfoffline/polling_osf_manager/polling.py

```diff
@@ -130,54 +174,37 @@ def make_local_remote_tuple_list(self, local_list, remote_list):
 
     @asyncio.coroutine
     def check_osf(self, remote_user):
-        logging.debug('check_osf')
-
         assert isinstance(remote_user, dict)
         assert remote_user['type'] == 'users'
 
         remote_user_id = remote_user['id']
+        nodes_url = api_url_for(USERS, related_type=NODES, user_id=remote_user_id)
 
-        # all_remote_nodes_url = 'https://staging2.osf.io:443/api/v2/users/{}/nodes/'.format(remote_user_id)
-        all_remote_nodes_url = api_url_for(USERS, related_type=NODES, user_id=remote_user_id)
-        while self._keep_running:
-
-            # get remote top level nodes
-            try:
-                remote_top_level_nodes = yield from self.osf_query.get_top_level_nodes(all_remote_nodes_url)
-            except (aiohttp.errors.ClientConnectionError, aiohttp.errors.ClientTimeoutError,
-                    concurrent.futures._base.TimeoutError):
-                # NOTE: can't work with partial list! That would suggest that nodes were created online.
-                AlertHandler.warn("Bad Internet Connection")
-                # waits till the end of a sleep to stop. thus can make numerous smaller sleeps
-                for i in range(POLL_DELAY):
-                    yield from asyncio.sleep(1)
-                continue
-
+        while True:
```

Comment:
Just to be sure: previously we relied on `keep_running` to stop this loop, but now we use the future to cancel/cleanup instead?